tests/features/repo.feature

```diff
@@ -151,3 +151,37 @@ Feature: Subcommand: dab repo
 		And the directory "~/dab/dotfiles14/.git/modules/dotfiles" should exist
 
 		And the file "~/dab/dotfiles14/dotfiles/.git" should exist
+
+	Scenario: Can list repos clean status
+		Given a file named "~/.config/dab/repo/dotfiles15/url" with:
+		"""
+		https://github.com/Nekroze/dotfiles.git
+		"""
+		And I successfully run `dab repo add dotfiles16 https://github.com/Nekroze/dotfiles.git`
+
+		And I successfully run `dab repo add dotfiles17 https://github.com/Nekroze/dotfiles.git`
+
+		And I successfully run `echo "not clean" > ~/dab/dotfiles17/not_clean.txt`
+
+		When I successfully run `dab repo list`
+
+		Then the output should match /^REPO\s*|\s*STATUS\s*|\s*CLEAN\s*/
+		And the output should match /^dotfiles15\s*|\s*not downloaded\s*|\s+/
+		And the output should match /^dotfiles16\s*|\s*downloaded\s*|\s*✓/
+		And the output should match /^dotfiles17\s*|\s*downloaded\s*|\s*✗/
+
+	Scenario: Can list repos UpToDate status
+		Given a file named "~/.config/dab/repo/dotfiles18/url" with:
+		"""
+		https://github.com/Nekroze/dotfiles.git
+		"""
+		And I successfully run `dab repo add dotfiles19 https://github.com/Nekroze/dotfiles.git`
+		And I successfully run `dab repo add dotfiles20 https://github.com/Nekroze/dotfiles.git`
+		And I successfully run `sh -c "cd ~/dab/dotfiles20" && git reset --hard HEAD~1`
```

Comment:
Have a look at the tests/features/README.md for links to the rest of the steps Aruba can provide, this might be easier done with the cd step

Reply:
That would be a lot easier, but the cd step does not find the proper paths, sadly.